examples/MODULE.bazel

```diff
@@ -6,10 +6,25 @@ local_path_override(
     path = "..",
 )
 
+bazel_dep(name = "contrib_rules_jvm", version = "0.26.0")
 bazel_dep(name = "platforms", version = "0.0.10")
 bazel_dep(name = "rules_cc", version = "0.0.9")
 bazel_dep(name = "rules_java", version = "7.3.1")
+bazel_dep(name = "rules_jvm_external", version = "6.1")
```

Comment:
R_j_e unfortunately has a hard requirement on Bazel 7. 

Maybe we can use a `single_version_override` to patch out the `use_repo_rule` usage? I could also give that a try, I wouldn't want to lose Bazel 6 coverage yet.

Reply:
ah, I am not familiar enough with bzlmod yet to figure that out but given there are just 4 artifacts needed for running the junit tests, I could just add simple `http_jar`s or whatever for them. Will tackle that tomorrow

Reply:
I pushed a commit to restore Bazel 6 compatibility by using a fork of r_j_e. I just removed everything that wasn't compatible, so I hope that doesn't break elsewhere.